Beam/ElasticBeam.C

```diff
@@ -717,7 +717,7 @@ bool ElasticBeamNorm::evalInt (LocalIntegral& elmInt, const FiniteElement& fe,
 
     // L2-norm of displacement error
     U -= Uh;
-    pnorm[ip++] += U*U*fe.detJxW;
+    pnorm[ip] += U*U*fe.detJxW;
```

Comment:
Maybe just skip the `ip` variable here, since no loop. Replace by `pnorm[0]`, `pnorm[1]` and `pnorm[2]`, respectively.
Besides, the first assignement (of `pnorm[0]`) should also have been an `+=` operation, like the others. Will/should not matter in real, since this beam element only uses one integration point, only to be consistent.

Reply:
done and done.